Makefile

```diff
@@ -1,9 +1,17 @@
 NAME    := unsee
 VERSION := $(shell git describe --tags --always --dirty='-dev')
+GO      := GO15VENDOREXPERIMENT=1 go
+PROMU   := $(GOPATH)/bin/promu
+pkgs     = $(shell $(GO) list ./... | grep -v -E '/vendor/')
```

Comment:
where is this used?

Reply:
Those are rather intrusive changes to the current workflow and they are not in scope for the ticket this aims to solve